src/babel-benchmark.js

```diff
@@ -2,14 +2,27 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-const Babel = require("@babel/standalone");
+const Babel = require("@babel/core");
 const babylon = require("babylon");
 const fs = require("fs");
 
 const payloads = [
   {
     name: "vue.runtime.esm-nobuble-2.4.4.js",
-    options: { presets: ["es2015"], sourceType: "module" }
+    options: {
+      presets: [
+        [
+          "@babel/preset-env",
+          {
+            targets: {
+              browsers: ["last 2 versions"],
+              node: 4 // https://github.com/nodejs/Release#release-schedule
```

Comment:
Targeting node 4 will result in different output - I think this is a situation where we might want different tests to cover different situations

- compiling to old browsers (IE)
- compiling to LTS node
- compiling to latest chrome/node/v8 master (that would basically mean only proposals and tests the baseline speed of startup)

kind of like what I mentioned in https://github.com/v8/web-tooling-benchmark/issues/27

Reply:
I agree with this and everything else you say in #27, but let’s do this in a follow-up PR.